controllers/utils/utils.go

```diff
@@ -27,6 +28,8 @@ var (
 )
 
 const (
+	defaultIstioControlPlaneName             = "data-science-smcp"
+	defaultMeshNamespace                     = "istio-system"
```

Comment:
Use the existing constant, if possible: https://github.com/opendatahub-io/odh-model-controller/blob/b1e69dd691bff8144ed07e703399b37af48ea0e6/controllers/constants/constants.go#L21

Reply:
changed, thanks.